src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +124,37 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        /**
+         * Represents the 'Export' button	
+         * 
+         */
+        JPanel buttonPanel = new JPanel();
+        buttonPanel.add(new JButton(new AbstractAction(Constant.messages.getString("fuzz.options.button.export")) {
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				WritableFileChooser chooser = new WritableFileChooser();
+				chooser.setSelectedFile(new File("Untitled.csv"));
+				if (chooser.showSaveDialog(null) == JFileChooser.APPROVE_OPTION) {
+					String file = chooser.getSelectedFile().toString();
+					if (!file.endsWith(".csv")) file += ".csv";
+					try (PrintWriter pw = new PrintWriter(new File(file))) {
+		        		int count = currentFuzzer.getMessagesModel().getRowCount();
+		        		for (int i = 0; i < count; i++) {
+		        			pw.println(currentFuzzer.getMessagesModel().getEntry(i).toString());
+		                }
+		        		JOptionPane.showMessageDialog(null, Constant.messages.getString("fuzz.options.button.export.message"));
+		        	} catch(Exception ex) {
+		        		logger.error(ex.getMessage(), ex);

```

Comment:
The user should be informed that an error occurred, otherwise there's no indication that the export already finished.